src/app/content/highlights/components/DisplayNote.spec.tsx

```diff
@@ -69,6 +70,27 @@ describe('DisplayNote', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('matches snapshot when click outside DisplayNote', () => {
+    const onClickOutside = jest.spyOn(onClickOutsideModule, 'useOnClickOutside');
+    onClickOutside.mockReturnValue();
```

Comment:
It looks like this line is causing an error that shows up later. I took it out and everything was fine.